src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -941,25 +941,17 @@ public static bool SupportsOrientationChanges()
 
 		#region Event Loop
 
-		public static void RunLoop(Game game)
+		static bool osxUseSpaces;
```

Comment:
This can be put at the top next to the other static constants, probably as

```
private static readonly bool OSXUseSpaces = (
    SDL.SDL_GetPlatform().Equals("Mac OS X") && // Prevents race with OSVersion
    SDL.SDL_GetHintBoolean(SDL.SDL_HINT_VIDEO_MAC_FULLSCREEN_SPACES, 1) == SDL.SDL_bool.SDL_TRUE
);
```